cmd/kubernetes/kubernetes.go

```diff
@@ -133,7 +133,7 @@ func runControllerManager(machineList []string, cl *client.Client, nodeMilliCPU,
 	go util.Forever(func() { endpoints.SyncServiceEndpoints() }, time.Second*10)
 
 	controllerManager := controller.NewReplicationManager(cl)
-	controllerManager.Run(10 * time.Second)
+	controllerManager.Run(controller.TickSecs)
```

Comment:
That's very confusing, since this function actually accepts the standard Go duration, which is in nanoseconds. Either TickDuration or TickSecs*time.Second would be clearer.